administrator/language/de-DE/com_menus.ini

```
@@ -75,7 +75,7 @@ COM_MENUS_ITEM_FIELD_ANCHOR_REL_LABEL="Rel-Attribut für Menülink"
 COM_MENUS_ITEM_FIELD_ASSIGNED_LABEL="Menü"
 COM_MENUS_ITEM_FIELD_ASSOCIATION_NO_VALUE="- Kein verknüpfter Menüpunkt definiert -"
 COM_MENUS_ITEM_FIELD_BROWSERNAV_LABEL="Zielfenster"
-COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC="Die Menüeinträge auswählen, die unter diesem Container angezeigt oder versteckt werden sollen. Wenn es keine Menueinträge die angezeigt werden sollen gibt, dann wird dieser Container ausgeblendet.<br />Bei der Installation einer neuen Komponente wird diese standardmäßig angezeigt, bis diese hier explizit versteckt wird."
+COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC="Die Menüeinträge auswählen, die in diesem Container angezeigt oder versteckt werden sollen. Wenn es keine Menueinträge gibt, die angezeigt werden sollen, dann wird dieser Container ausgeblendet.<br />Bei der Installation einer neuen Komponente wird diese standardmäßig angezeigt, bis diese hier explizit versteckt wird."
```

```suggestion
COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC="Die Menüeinträge auswählen, die in diesem Container angezeigt oder versteckt werden sollen. Wenn es keine Menueinträge gibt, die angezeigt werden sollen, dann wird dieser Container ausgeblendet.<br />Bei der Installation einer neuen Komponente wird diese standardmäßig angezeigt, bis sie hier explizit versteckt wird."
```

Nachdem ich den englischen Text noch einmal gelesen habe, bin ich der Meinung, dass sich das letzte "diese" auf die Komponente bezieht. Man könnte mein eingefügtes "sie" auch durch "die Komponente" ersetzen, um die "Undeutlichkeit" zu beseitigen.

---

```suggestion
COM_MENUS_ITEM_FIELD_COMPONENTS_CONTAINER_HIDE_ITEMS_DESC="Die Menüeinträge auswählen, die in diesem Container angezeigt oder versteckt werden sollen. Wenn es keine Menueinträge gibt, die angezeigt werden sollen, dann wird dieser Container ausgeblendet.<br />Bei der Installation einer neuen Komponente wird diese standardmäßig angezeigt, bis die Komponente hier explizit versteckt wird."
```